src/main/java/org/ladocuploader/app/submission/actions/ValidateBirthdate.java

```diff
@@ -0,0 +1,33 @@
+package org.ladocuploader.app.submission.actions;
+
+import formflow.library.data.FormSubmission;
+import formflow.library.data.Submission;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.regex.Pattern;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+
+@Slf4j
+@Component
+public class ValidateBirthdate extends VerifyDate {
+  private final String INPUT_NAME = "birthDay";
+
+  @Override
+  public Map<String, List<String>> runValidation(FormSubmission formSubmission, Submission submission) {
+    Map<String, List<String>> errorMessages = new HashMap<>();
+    Map<String, Object> inputData = formSubmission.getFormData();
+    String day = (String) inputData.get("birthDay");
+    String month = (String) inputData.get("birthMonth");
+    String year = (String) inputData.get("birthYear");
+
+    if (!Pattern.matches("\\d{4}",year)) {
+      errorMessages.put(INPUT_NAME, List.of("Make sure to provide a date in MM/DD/YYYY OR M/D/YYYY."));
```

Comment:
These error messages (line 26 & 28) should probably be message source properties, right?

Reply:
Or MessageSource is needed here to translate them before handing them back?

Reply:
Also, perhaps this message should just be about year since that's what it's checking?  The three fields are separate in the ui, so if they get a message about the full birthdate but the issue is the Year, it might be confusing.